### PR TITLE
Fixed link definitions on cloud/live/stage-prod-test.html

### PR DESCRIPTION
**Pull request by sergey-nechaev**
## Purpose of this pull request

- Fixed problems with link definition rendering
- Updated 2 URLs to avoid redirects

## Affected DevDocs pages

https://devdocs.magento.com/cloud/live/stage-prod-test.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by meker12** [comment]
running tests

File: src/cloud/live/stage-prod-test.md [hunk 1]
```diff
@@ -228,24 +228,25 @@ You can use the Magento Functional Testing Framework (MFTF) to complete function
...
-[Testing before changing DNS](https://docs.fastly.com/guides/basic-configuration/testing-setup-before-changing-domains
+[Testing before changing DNS]: https://docs.fastly.com/en/guides/testing-setup-before-changing-domains
 [Access info]: {{ site.baseurl }}/cloud/live/stage-prod-migrate-prereq.html#starter-urls
 [`Cache-Control: max-age`]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9
 [`Pragma`]: https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.32
 [Set up Fastly]: {{ site.baseurl }}/cloud/cdn/cloud-fastly.html
 [Fastly troubleshooting]: {{ site.baseurl }}/cloud/cdn/trouble-fastly.html
-[Magento Performance Toolkit]({{ site.mage2bloburl }}/{{ site.version }}/setup/performance-toolkit)
+[Magento Performance Toolkit]: {{ site.mage2bloburl }}/{{ site.version }}/setup/performance-toolkit
 [Magento application performance test]: {{ site.baseurl }}/cloud/env/variables-post-deployhtml#ttfb_tested_pages
```
Comment (meker12) [inline src/cloud/live/stage-prod-test.md:245]:
```suggestion
[Magento application performance test]: {{ site.baseurl }}/cloud/env/variables-post-deploy.html#ttfb_tested_pages
```

**Comment by meker12** [comment]
running tests

**Pull request merged**